src/test/java/com/wikia/webdriver/testcases/desktop/discussions/CreatingReplyTests.java

```diff
@@ -75,11 +75,11 @@ public void userOnDesktopCanCreateReplyOnPostDetailsPage() {
   // Testing methods
 
   private void userOnDesktopMustBeLoggedInToUseReplyCreator(final ReplyCreator replyCreator) {
-    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.REGISTER_PAGE);
+    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.LOGIN_PAGE);
   }
 
   private void userOnMobileMustBeLoggedInToUseReplyCreator(final ReplyCreator replyCreator) {
-    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.JOIN_PAGE);
+    anonymousUserOnReplyEditorClickIsRedirectedTo(replyCreator, MobileSubpages.LOGIN_PAGE);
   }
 
   private void anonymousUserOnReplyEditorClickIsRedirectedTo(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "anonymousUserOnReplyEditorClickIsRedirectedTo" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)